content/applications/hr/fleet/new_vehicle.rst

```diff
@@ -0,0 +1,146 @@
+  the drop-down menu, or type in a new company and click either :guilabel:`Create` or
+  :guilabel:`Create and Edit`.
+
+.. image:: new_vehicle/new-vehicle-type.png
+   :align: center
+   :alt: The new vehicle form, showing the vehicle tax section.
+
+Tax Info tab
+------------
+
+Fiscality
+~~~~~~~~~
+
+- :guilabel:`Horsepower Taxation`: Enter the amount that is taxed based on the size of the vehicles
+  engine. This is determined by local taxes and regulations, and varies depending on the location.
+  It is recommended to check with the accounting department to ensure this value is correct.
+- :guilabel:`Disallowed Expense Rate`: Enter the amount of non-deductible expenses for the vehicle.
+  This amount is not counted towards any deductions on a tax return or as an allowable expense
+  when calculating taxable income. It is recommended to check with the accounting department to
+  ensure this value is correct.
+- :guilabel:`Start Date`: Enter the date these two values will affect the taxes, using the
+  drop-down calendar module.
+
+Contract
+~~~~~~~~
+
+- :guilabel:`First Contract Date`: Select the start date for the vehicle's first contract using
+  the drop-down calendar module. Typically this is the day the vehicle is purchased or leased.
+- :guilabel:`Catalog Value (VAT Incl.)`: Enter the MSRP (Manufacturer's Suggested Retail Price) for
+  the vehicle at the time of purchase or lease.
+- :guilabel:`Purchase Value`: Enter the purchase price or the value of the lease for the vehicle.
+- :guilabel:`Residual Value`: Enter the current value of the vehicle.
+
+.. image:: new_vehicle/new-vehicle-tax.png
+   :align: center
+   :alt: The new vehicle form, showing the vehicle tax section.
+
+Model tab
+---------
+
+Model
+~~~~~
+
+- :guilabel:`Model Year`: Enter the year the vehicle was manufactured.
+- :guilabel:`Transmission`: Select the transmission type from the drop-down menu, either
+  :guilabel:`Manual` or :guilabel:`Automatic`.
+- :guilabel:`Color`: Enter the color of the vehicle.
+- :guilabel:`Seats Number`: Enter how many passengers the vehicle can accommodate.
+- :guilabel:`Doors Number`: Enter the number of doors the vehicle has.
+- :guilabel:`Trailer Hitch`: Check this box if the vehicle has a trailer hitch.
+
+Engine
+~~~~~~
+
+- :guilabel:`Horsepower`: Enter the vehicle's horsepower in this field.
+- :guilabel:`Power`: If the vehicle is electric or hybrid, enter the power the vehicle uses in
+  kilowatts (kW).
+- :guilabel:`Fuel Type`: Select the type of fuel the vehicle uses form the drop-down menu. The
+  options are :guilabel:`Diesel`, :guilabel:`Gasoline`, :guilabel:`Hybrid Diesel`, :guilabel:`Hybrid
+  Gasoline`, :guilabel:`Plug-in Hybrid Diesel`, :guilabel:`Plug-in Hybrid Gasoline`,
+  :guilabel:`CNG`, :guilabel:`LPG`, :guilabel:`Hydrogen`, or :guilabel:`Electric`.
+- :guilabel:`CO2 Emissions`: Enter the carbon dioxide emissions the vehicle produces in grams per
```

Comment:
average?

Reply:
Thank you, yes, and it's from the manufacturer/dealership- not a changeable number. I added that as well.